docs/architecture/mp-components.md

```diff
@@ -0,0 +1,17 @@
+# Management Plane Components
+
+## Overview
+
+This document describes the Management Plane's components.
+
+## Components
+
+Management Plane consists of three separate components. Applications, Agents, and UIs can communicate with the Gateway component or Connector component.
```

Comment:
Please add UI, ServiceMesh and RuntimeProvisioner as an additional components into Management Plane. They are redeemable. 

All existing ones belongs to Compass, and Compass itself belongs to Management Plane. Please see https://github.com/kyma-incubator/compass/pull/31 for details.